starter-code/lab-mongoose-movies/views/celebrities-page.hbs

```diff
@@ -0,0 +1,29 @@
+<h2>Celebrities Page</h2>
+ <a href="/celebrities/new">
+     <button>
+        Add New Celebrity
+     </button>
+  </a>
+  <br>
+  
+
+{{#each celebrityList}} 
+ <ul>  
+   <a href="/celebrities/{{this._id}}">
+       <li>{{this.name}}</li>
+   </a>
+   
+   <li>{{this.occupation}}</li>
```

Comment:
Notice that you are creating more `li`s, so it looks like each celebrity is three separate celebrities. Maybe it's a better UX to just stick with the name?